### PR TITLE
Add match-series

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@din14970

**Comment by jan-janssen** [comment]
@din14970 Can you briefly comment on this pull request and confirm that you join me in maintaining this recipe?

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/match-series/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+
+test:
+  commands:
+    - ${PREFIX}/bin/projects/electronMicroscopy/matchSeries --help
+
+about:
+  home: https://github.com/berkels/match-series
+  license: CDDL
+  license_family: OTHER
```
Comment (chrisburr) [inline recipes/match-series/meta.yaml:38]: The SPDX identifier has a version, either `CDDL-1.0` or `CDDL-1.1`: https://spdx.org/licenses/

You can then remove `license_family`
```suggestion
  license: CDDL-X.Y
```

File: recipes/match-series/build.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      -DUSE_GRAPE=0 \
+      -DUSE_C++11=1 \
+      -DGENERATE_INCLUDE_TEST=0 \
+      -DUSE_TIFF=1 \
+      -DUSE_CIMG=0 \
+      -DTEST_OPENSOURCE_SELECTION=0 \
+      -DUSE_OPENGL=0 \
+      -DUSE_METIS=0 \
+      quocmesh
+make
```
Comment (chrisburr) [inline recipes/match-series/build.sh:31]:
```suggestion
make -j${CPU_COUNT}
```

File: recipes/match-series/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{% set name = "match-series" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/berkels/{{ name }}/archive/master.tar.gz
```
Comment (chrisburr) [inline recipes/match-series/meta.yaml:9]: Can you try and get upstream to tag a version?

Comment (jan-janssen) [reply recipes/match-series/meta.yaml:9]: I create a request upstream https://github.com/berkels/match-series/issues/2

**Comment by din14970** [comment]
I'm ok to be listed here as maintainer

File: recipes/match-series/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{% set name = "match-series" %}
+{% set version = "0.0.1" %}
```
Comment (chrisburr) [inline recipes/match-series/meta.yaml:2]:
```suggestion
{% set version = "1.0.0" %}
```

File: recipes/match-series/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/berkels/{{ name }}/archive/f032293b5a3259d5bc30e82d5fd580c2e525c9e4.tar.gz
```
Comment (chrisburr) [inline recipes/match-series/meta.yaml:9]:
```suggestion
  url: https://github.com/berkels/{{ name }}/archive/{{ version }}.tar.gz
```

File: recipes/match-series/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 72204dac55d529165836c7503dfb388482082840ef9b7f9aeda1f29acf502811
```
Comment (chrisburr) [inline recipes/match-series/meta.yaml:10]:
```suggestion
  sha256: 9e74ca6bc58571065f30f99307f8912009fa6b62be10f24d3f5ddc5b2361ce48
```

**Comment by jan-janssen** [comment]
@chrisburr I updated the recipe to use the GitHub release.

**Pull request merged**